critiquebrainz/frontend/views/review.py

```diff
@@ -365,7 +366,7 @@ def hide(id):
 
     form = AdminActionForm()
     if form.validate_on_submit():
-        db_review.hide(review["id"])
+        db_review.set_hidden_state(review["id"], is_hidden="True")
```

Comment:
The `is_hidden` argument should be a `bool`. Pretty sure you can pass it straight to psycopg too.